src/frontend/src/components/creacionCuentas/ctaCorriente.vue

```diff
@@ -43,16 +35,34 @@
       </v-container>
     </v-app>
   </div>
+  <footer class="footer">
+    <div class="brand">
+      <p>Marca Registrada &reg;</p>
+    </div>
+  </footer>
+
+
 </template>
 
-<script></script>
+<script>
+export default {
+  methods: {
+    redireccionarAHome() {
+      this.$router.push('/home')
+    }
+  }
+}
+</script>
 
 <style scoped>
 .pagina {
   width: 100%;
 }
+
 .titulo {
   text-align: center;
+  font-size: 50px;
+  color: #0039A6;
 }
```

Comment:
Sugerencia: añadir negrita al título. Se ve mejor el título en negrita.

```suggestion
.titulo {
  text-align: center;
  font-size: 50px;
  color: #0039A6;
  font-weight: bold;
}
```